build-system/global.d.ts

```diff
@@ -0,0 +1,13 @@
+declare global {
+  interface Error {
```

Comment:
Maybe @ types/node will help here too instead of defining a custom interface.

Reply:
Unfortunately neither showStack nor status is not an attribute of Error so we either need this or a custom Error type.

Reply:
Ah I see.. looks like it's a gulp type. I don't know about status but I found showStack here https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/plugin-error/index.d.ts#L13

Reply:
Yep, `showStack` is for `gulp`, and should go away when we adopt a more lightweight task runner.

`status` is something we set in `exec.js`, so it might be possible to use a different mechanism:

https://github.com/ampproject/amphtml/blob/29fdb586db86f940a085d35bc7d8a2a16b45e53c/build-system/common/exec.js#L80-L97

Reply:
I would rather not alter the existing error message handling system for now. Given that those are both properties that we do actually use this type extension makes sense to me, but should remove these properties once the code using them is gone.

Reply:
Ok that's far, how about adding a TODO in global.d.ts to delete it

Reply:
Done